src/app/convert/css-scss/components/ResultBox.jsx

```diff
@@ -0,0 +1,14 @@
+export default function ResultBox({
+  to,
+  resultSnippet,
+  isDarkMode,
+}) {
+  return (
+    <textarea
+      className={`w-full shadow-inner p-4 border-0 ${isDarkMode ? "bg-gray-700 text-white" : "bg-white text-black"}`}
```

Comment:
```suggestion
      className={` resize-none w-full shadow-inner p-4 border-0 ${isDarkMode ? "bg-gray-700 text-white" : "bg-white text-black"}`}
```
To disable the resize